### ISSUE TITLE
Polygon Icon Hover State Problem

### ISSUE THREAD

**Issue by bulentAkgul** [nhn/tui.image-editor#360]
I changed all icons' hover colors, but it doesn't work for poligon icon. It uses this:
````html
<use xlink:href="/images/icon-a.svg?1f2f3ed047d2f26b5fdaae448ed70c56#icon-a-ic-icon-polygon" class="active"></use>````

As you can see it's grabbing the icon from icon-a set, but the others from icon-c.

**Comment by jinwoo-kim-nhn** [comment]
There is a bug. I'll check.

**Comment by jinwoo-kim-nhn** [comment]
It seems to have been fixed in v3.9.0. Please upgrade and check.
